cli/global.go

```diff
@@ -57,6 +58,11 @@ func ConfigureGlobals(app *kingpin.Application) {
 		Envar("AWS_VAULT_KEYCHAIN_NAME").
 		StringVar(&GlobalFlags.KeychainName)
 
+	app.Flag("libsecret-collection", "Name of libsecret collection to use, if it doesn't exist it will be created").
```

Comment:
The backend is called `secret-service` So might make sense for this to be `--secret-service-collection`?

Reply:
Sure, I guess. I was going by preexisting variable names, but backend name is probably better for user-provided parameters as libsecret is not mentioned anywhere.